src/test/java/school/redrover/Pipeline1Test.java

```diff
@@ -453,6 +454,30 @@ public void testStageColumnHeader() {
             Assert.assertEquals(actualResult, expectedResult);
         }
     }
+
+    @Test
+    public void testPipelineProjectCanBeReenabled() {
+        String pipelineName = "My new cool disable pipeline project";
+
+        HomePage homePage = new HomePage(getDriver());
+        String statusForJobNameGiven = homePage
+            .clickNewItem()
+            .setItemName(pipelineName)
+            .selectPipelineAndClickOk()
+            .clickSaveButton()
+            .clickDisableEnableProjectButton()
+            .clickLogo()
+            .getStatusFor(pipelineName);
+        Assert.assertEquals(statusForJobNameGiven, "Disabled");
+
+        String statusForJobNameGivenAfterReenable = homePage
```

Comment:
```suggestion
        String statusForJobNameGivenAfterReenable = new HomePage(getDriver())
```

Это уже не та HomePage которую ты создал раньше, она уже была перезагруженна.

Reply:
Страница перезагружена, но я так понимаю что я могу к ней обращаться через тот же объект, так как элементы инициализированные через PageFactory "looked up each and every time a method is called upon it"[ ](https://www.selenium.dev/selenium/docs/api/java/org/openqa/selenium/support/PageFactory.html)
Или я что-то не так понимаю?

Reply:
Нет не можешь, выше я написал почему нет и почему у тебя работает хотя должна быть ошибка.